recipes/mmcv/meta.yaml

```diff
@@ -0,0 +1,130 @@
+{% set version = "1.5.3" %}
+{% set number = 0 %}
+{% set proc_type = "cuda" if cuda_compiler_version != "None" else "cpu" %}
+{% if cuda_compiler_version != "None" %}
+{% set number = number + 200 %}
+{% endif %}
+
+package:
+  name: mmcv-recipe
+  version: {{ version }}
+
+source:
+  url: https://github.com/open-mmlab/mmcv/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 650929f52f1d0b2bae1bdaaa874e1e6dfdfa15f40bab5c366330488a9949ddd0
+build:
+  number: {{ number }}
+  skip: true  # [win]
+
+outputs:
+  - name: mmcv-full
+    build:
+      string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+      string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                      # [cuda_compiler_version == "None"]
+      detect_binary_files_with_prefix: false
+      run_exports:
+        - {{ pin_subpackage('mmcv-full', max_pin='x.x') }}
+      missing_dso_whitelist:
+        # conda-forge::pytorch provides these libs
+        - "$RPATH/libc10_cuda.so"
+        - "$RPATH/libtorch_python.so"
+        - "$RPATH/libtorch_cpu.so"
+        - "$RPATH/libc10.so"
+        - "$RPATH/libc10_cuda.dylib"
+        - "$RPATH/libtorch_python.dylib"
+        - "$RPATH/libtorch_cpu.dylib"
+        - "$RPATH/libc10.dylib"
+    script: build_mmcv.sh  # [not win]
+    requirements:
+      build:
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        # this adds matching cuda requirement to run deps using __cuda package
+        - {{ compiler('cuda') }}    # [cuda_compiler_version != "None"]
+        - ninja
+      host:
+        - python
+        - pip
+        - requests
+        - future
+        - six
+        - typing
+        - pkg-config  # [unix]
+        - typing_extensions
+        - pytorch ={{ pytorch }}={{ proc_type }}*
+        - torchvision
+        - setuptools
+      run:
+        # intentionally skipping opencv and pytorch (following upstream vendor)
```

Comment:
`mmcv-full` depends on those two, but upstream vendor doesn't put them as runtime dependencies in pip packaging. I would imagine that it's because:
1) you can use parts of the library without those heavy deps (especially without opencv)
2) opencv with 4 distribution options under Pip (contrib/not-contrib, headless/not-headless) makes it a mess to depend on a single one. I.e. if your package depends on `headless` version and user has already normal one installed - pip will still install `headless` on top of that. I think the same will happen with conda, i.e. if users uses FastAI's headless version which is quite popular.